### ISSUE TITLE
Remake theme using Bootstrap (or other UI framework)

### ISSUE THREAD

**Issue by JakeBooher** [ezrpg/ezRPG-1.2.x#50]
Will help usability by making it responsive. Will probably use Bootstrap 4 for it

**Comment by uaktags** [comment]
Personally i hate Bootstrap for being so dang bloated. But it's by far probably one of the best options.

**Comment by JakeBooher** [comment]
You don't have to include all of bootstrap, you can compile it with only what you need

**Comment by hubbyist** [comment]
:+1: Bootstrap is so dang bloated

**Comment by uitgewis** [comment]
This would be a good candidate for a third-party plugin/theme.

**Comment by johndevsvensson** [comment]
I agree on a new default theme. Bootstrap or not, shouldn't really matter as themes imo should be detached from ezRPG and you could make a theme with any CSS/JavaScript framework/libraries you like.
Though could probably make sense to have CSS Coding guidelines as well as for JavaScript.

**Comment by JakeBooher** [comment]
The default needs to, at least, be responsive
